### ISSUE TITLE
Почему не отображаются картинки на сайте?

### ISSUE THREAD

**Issue by nadindindin** [GdeSlon/wp-affiliate-shop#21]
Здравствуйте! Я установила плагины, как указано в инструкции «GdeSlon Affiliate Shop» и плагин «WooCommerce excelling eCommerce», все настроила. Но при выгрузки товаров, на сайте не отображаются картинки (изображение товара) и ко всему этому не могу перейти на страницу товара (На этой странице обнаружена циклическая переадресация).
Подскажите почему так происходит и как это исправить?
Вот страница сайта: http://mirpodarkovonline.ru/
Спасибо, жду ответ!
С уважением, Надежда.

**Comment by aiiddqd** [comment]
Попробуйте пересохранить постоянные ссылки в параметрах консоли WordPress. Просто зайти и нажать Сохранить.

**Comment by nadindindin** [comment]
Нажать просто сохранить? У меня в консоли постоянных ссылок вот как настроено http://www.floomby.ru/s2/KefU9p
Как вы сказали, я нажала сохранить, но ничего не поменялось(
По прежнему нет изображений и не переходит на страницу товара(((
http://mirpodarkovonline.ru/shop/

**Comment by aiiddqd** [comment]
Попробуйте поменять параметры.
Сделать открытие через товар

**Comment by nadindindin** [comment]
Ура, получилось зайти на страницу товара!!!
Но по прежнему нет картинок(((
http://www.floomby.ru/s2/BefUzu

**Comment by aiiddqd** [comment]
Это сложнее. Без диагностики сложно назвать причину.
Вот тут ее можно заказать http://systemo.biz/diagnostika-wordpress/

**Comment by nadindindin** [comment]
Это платно?

**Comment by aiiddqd** [comment]
Да

**Comment by nadindindin** [comment]
сколько?

**Comment by aiiddqd** [comment]
1,5 т.р.

**Comment by nadindindin** [comment]
а другой путь есть? Может я самостоятельно могу настроить? только подскажите как и где?!

**Comment by aiiddqd** [comment]
да, надо включить режим отладки, изучить код плагина, найти ошибку в логике механизма и ее причины.

**Comment by nadindindin** [comment]
а как можно без плагина сделать выгрузку с "Где слон"?

**Comment by aiiddqd** [comment]
если хотите чтобы вам кто то помог с вашими вопросами, то следует соблюдать этикет общения в сети.
новый вопрос = новая тема.
я не знаю ответа на него, а те кто знают его могут просто не заметить

**Comment by nadindindin** [comment]
Спасибо за ответ! Исправлюсь...

**Comment by letions** [comment]
Здравствуйте! Помогу загрузить товары с прайс-листа на OpenCart или Ostore — смотрятся сайты намного солиднее на данных CMS. Вот пример — http://myopencart.com/ .
В карточке товара будут помимо основного фото — дополнительные фото, актуальная цена, заголовок, наличие товара, описание, характеристики с атрибутами — все как на сайте рекламодателя!
Создам интернет-магазин с Вашим прайсом.
Пишите на letions@mail.ru
